client/my-sites/media-library/header.jsx

```diff
@@ -80,11 +80,11 @@ class MediaLibraryHeader extends React.Component {
 					site={ site }
 					filter={ filter }
 					onAddMedia={ onAddMedia }
-					className="button is-compact"
+					className="media-library__upload-button button is-compact"
 				>
 					<Gridicon icon="add-image" />
-					<span className="is-desktop">
-						{ this.props.translate( 'Add New', { context: 'Media upload' } ) }
+					<span className="media-library__upload-button-label">
+						{ this.props.translate( 'Add new', { context: 'Media upload' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **12** times:
`translate( 'Add new', { context: 'new image'} )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/button-sentence-case)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `add new media`
- `Block`
- `Customize Changeset`
- `event`
- `file`
- `link`
- `Media upload`
- `page`
- `plugin`
- `post`
- `site`
- `Template`
- `Template Part`
- `theme`
- `user`

 Would it make sense to reuse one of the above?